x-pack/plugins/watcher/public/application/sections/watch_edit_page/components/json_watch_edit/json_watch_edit_simulate.tsx

```diff
@@ -429,6 +420,16 @@ export const JsonWatchEditSimulate = ({
           })}
         </EuiButton>
       </EuiForm>
+
+      <SimulateWatchResultsFlyout
```

Comment:
I moved this flyout to the bottom of the file because this location mirrors the location of the flyout in the UX. You see it after you see the form and the other stuff.